tests/math/test_functions.py

```diff
@@ -1871,7 +1961,7 @@ def test_autograd(self):
             np.array([[x, 1.2 * y], [x ** 2 - y / 3, -x / y]]),
         ]
         f = lambda x, y: fn.block_diag(tensors(x, y))
-        x, y = 0.2, 1.5
+        x, y = np.array([0.2, 1.5], requires_grad=True)

```

Comment:
This is to remove the same warning as above (i.e. explicit trainability marking in Autograd)